content/docs/tasks/telemetry/metrics/collecting-metrics/index.md

```diff
@@ -1,32 +1,33 @@
 ---
-title: Collecting Metrics and Logs
-description: This task shows you how to configure Istio to collect metrics and logs.
-weight: 20
+title: Collecting Metrics
+description: This task shows you how to configure Istio to collect and customize metrics.
+weight: 10
 keywords: [telemetry,metrics]
 aliases:
     - /docs/tasks/metrics-logs.html
+    - /docs/tasks/telemetry/metrics-logs/
 ---
 
 This task shows how to configure Istio to automatically gather telemetry for
-services in a mesh. At the end of this task, a new metric and a new log stream
-will be enabled for calls to services within your mesh.
+services in a mesh. At the end of this task, a new metric will be enabled for
+calls to services within your mesh.
 
 The [Bookinfo](/docs/examples/bookinfo/) sample application is used
 as the example application throughout this task.
 
 ## Before you begin
 
-* [Install Istio](/docs/setup/) in your cluster and deploy an
+* [Install Istio](/docs/setup/kubernetes) in your cluster and deploy an
   application. This task assumes that Mixer is setup in a default configuration
   (`--configDefaultNamespace=istio-system`). If you use a different
   value, update the configuration and commands in this task to match the value.
 
-## Collecting new telemetry data
+## Collecting new metrics
 
 1.  Create a new YAML file to hold configuration for the new metric and log
     stream that Istio will generate and collect automatically.
 
-    Save the following as `new_telemetry.yaml`:
+    Save the following as `new_metrics.yaml`:
 
     {{< text yaml >}}
```

Comment:
Add downloadas annotation as in the other file.

Reply:
Same comment as above.